### ISSUE TITLE
delegate team multisig to one wallet

### ISSUE THREAD

**Issue by pentcle** [pentaclexyz/strt-cred#20]

**Comment by pentcle** [comment]
This looks potentially promising

https://docs.ssx.id/

![Image](https://user-images.githubusercontent.com/80323528/224293037-c8c5321d-8b6a-4cca-bced-239c71a7c25d.png)

**Comment by ncerovac** [comment]
yep, just checked, this is doable with config https://docs.ssx.id/configuring-ssx

so this should be ready for dev work?

**Comment by ncerovac** [comment]
update: this for now is not the best solution since it not solving our usecase. 

we agreed for now to drop this func
